### PR TITLE
First PR: icons change PR1

### PR DESCRIPTION
**Pull request by swetha571**
# Type of change? 

- [x] Bug fix
- [ ] New feature
- [ ] Enhancement
- [ ] Breaking change (breaks backward compatibility)

# Description, Context, Motivation 
Icons changes
...

**Checklist:**

- [ ] All tests passed
- [ ] The code follows the code style and conventions of this project
- [ ] The change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: src/components/layout/leftNav/leftNav.js [hunk 1]
```diff
@@ -6,17 +6,17 @@ import lang from '../../../common/lang';
...
+import RulesIcon from '../../../assets/icons/RulesIcon.svg';
 import ContosoIcon from '../../../assets/icons/Contoso.svg';
 
 import './leftNav.css';
 
 const tabConfig = [
   { path: '/dashboard',    icon: DashboardIcon,    name: lang.DASHBOARD_LABEL },
   { path: '/devices',      icon: DevicesIcon,      name: lang.DEVICES },
-  { path: '/rulesActions', icon: RulesActionsIcon, name: lang.RULES },
+  { path: '/rulesActions', icon: RulesIcon, name: lang.RULES },
```
Comment (spryor) [inline src/components/layout/leftNav/leftNav.js]: `name` is no longer aligned

File: src/components/pages/dashboard/dashboard.js [hunk 2]
```diff
@@ -144,6 +144,7 @@ class DashboardPage extends Component {
             <div className="last-refreshed-text"> {`${lang.LAST_REFRESHED} | `} </div>
             <div className="last-refreshed-time">{this.state.lastRefreshed.toLocaleString()}</div>
             <div onClick={this.refreshData} className="refresh-icon icon-sm" />
+            <div className="time-icon icon-sm" />
             <Select {...selectProps} />
```
Comment (spryor) [inline src/components/pages/dashboard/dashboard.js]: `select` should be indented

File: src/components/pages/dashboard/dashboard.scss [hunk 3]
```diff
@@ -32,6 +33,12 @@
     color: #afb9c3;
   }
 
+  .time-icon{
```
Comment (spryor) [inline src/components/pages/dashboard/dashboard.scss]: `.time-icon[space]{`

File: src/components/pages/dashboard/dashboard.js (hunk 2 above)
Comment (swetha571) [reply src/components/pages/dashboard/dashboard.js]: Its indented.
image

**Pull request merged**